vue/components/ui/molecules/form-input/form-input.vue

```diff
@@ -53,6 +55,12 @@
     flex: 1;
 }
 
+.form-input .header.tooltip {

```

Comment:
Don't need the extra class, use this selector: `form-input .tooltip > .header`.

Also, isn't the color `#57626e` missing?

I think this is what the design wanted:
```
    text-decoration: underline #57626e dashed;
    text-underline-offset: 3px;
```

Reply:
Nice catch! Updated :)